docs/api/aerial/contract/__init__.md

```diff
@@ -22,7 +22,8 @@ Ledger contract.
 def __init__(path: Optional[str],
              client: LedgerClient,
              address: Optional[Address] = None,
-             digest: Optional[bytes] = None)
+             digest: Optional[bytes] = None,
+             schema_path: Optional[str] = None)
```

Comment:
Is this added manually? If so, you can run the script `scripts/generate_api_docs.py` to automatically update API doc.

Reply:
I checked and the API docs are up-to-date